### PR TITLE
Feat/search labels

### LINKED ISSUE #117: `labels`での検索一覧画面の実装
### 主題

`/search?labels=blog,sns`でコンポーネントの検索一覧を表示する

### 説明

- `i18n/content.en.json`があると仮定して、URLのクエリから`labels`でコンポーネントを絞り込みして`category`の画面のようにコンポーネントの一覧を表示する。
- `?labels`には複数にパラメタが設定されている場合はその条件で絞り込みをし表示をする。

JSONデータ例

```json
[
  {
    "title": "ページのタイトル名",
    "type": "形式、component・category・category-group・label",
    "description": "説明",
    "labels": ["blog"],
    "slug": "/layouts/buttons/copy-button",
    "hierarchy": { 
       "category-group": "Layouts",
       "category": "Button",
       "component": "Copy button",
    }
  },
 {
    "title": "ページのタイトル名",
    "type": "形式、component・category・category-group・label",
    "description": "説明",
    "labels": ["blog"],
    "slug": "/layouts/buttons/",
    "hierarchy": { 
       "category-group": "Layouts",
       "category": "Button"
    }
  }
]
```

### この機能の作成に参加し、プルリクエストを作成しますか？

None

### PR DESCRIPTION
**Pull request by illionillion**
Closes #117

### CODE REVIEW

File: components/data-display/search-labels-result.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          alignItems={{ base: "end", sm: "stretch" }}
+          justifyContent="space-between"
+          gap={{ base: "md", sm: "0" }}
+          paddingBlock="md"
+        >
+          <HStack>
+            {searchResult.labels.map((v) => (
+              <Tag key={v}>{v}</Tag>
+            ))}
+          </HStack>
```
Comment (taroj1205) [inline components/data-display/search-labels-result.tsx:35]: I think this code could be rewritten like below:
```tsx
          <HStack gap="xs">
            {searchResult.labels.map((v) => (
              <Tag key={v} justifyContent="center">
                {v}
              </Tag>
            ))}
          </HStack>
```

**Review by taroj1205 — COMMENTED** [review]
I think we should be able to search with `category group name` and `category name` without adding labels in `metadata.json`.
Also, the changes made to `index.tsx` (e.g. `comments/comment/index.tsx`) is not required to be part of this PR.

File: components/data-display/search-labels-result.tsx (hunk 1 above)
Comment (illionillion) [reply components/data-display/search-labels-result.tsx:35]: I fixed it.

**Review by taroj1205 — DISMISSED** [review]
LGTM

**Pull request merged**